sql.go

```diff
@@ -320,7 +320,7 @@ FROM   (SELECT n.nspname
         FROM   pg_catalog.pg_class c, 
                pg_catalog.pg_constraint con, 
                pg_namespace n 
-        WHERE  c.oid = '"%[1]s"' :: regclass 
+        WHERE  c.oid = '%[1]s' :: regclass 
```

Comment:
Is this intentional?

Reply:
I had to remove it, since it leads to syntax error 

```
FATA[2020-02-27 19:53:58] Encountered error when getting constraints for table "public"."uppercaseTable" from database, err: ERROR #42602 invalid name syntax 
exit status 1
```

Reply:
Oh, seems I do not understand the code as "well" as I thought ;)

Reply:
Haha no worries, thanks for your time and the pull request :)